pkg/operation/botanist/component/vpnshoot/vpnshoot.go

```diff
@@ -57,28 +59,38 @@ const (
 type Interface interface {
 	component.DeployWaiter
 	component.MonitoringComponent
+	SetPodAnnotations(map[string]string)
```

Comment:
I don't think such function is needed in the interface. Let's remove it (I don't see it called anywhere)

Reply:
there is a field 
`{{- if .Values.podAnnotations }}
{{ toYaml .Values.podAnnotations | indent 8 }}
{{- end }}`
in the deployment yaml.

Reply:
I haven't looked into how to incorporate the function in botanist though.

Reply:
I have used it now. let me push the new commits once.